sal_interfaces/MTVMS/MTVMS_Telemetry.xml

```diff
@@ -41,4 +41,72 @@
       <Count>50</Count>
     </item>
   </SALTelemetry>
+  <SALTelemetry>
+    <Subsystem>MTVMS</Subsystem>
+    <EFDB_Topic>MTVMS_psd</EFDB_Topic>
+    <Description>VMS (1 sensor, 3 channels) accelerometer Power Spectral Densities (PSDs).</Description>
+    <item>
+      <EFDB_Name>timestamp</EFDB_Name>
+      <Description>Start timestamp of computed PSDs.</Description>
```

Comment:
Consider:
Starting time of data acquisition (TAI unix seconds) used to compute PSDs.

Reply:
Changed, thanks.